docs/install.md

```diff
@@ -1,38 +1,64 @@
 # Installation
 
-We recommend **the easy way** if you are running your own server.
+There are many options for installing Sandstorm with various trade-offs. Choose the one that is most comfortable for you.
 
-## Installing the Easy Way
+Sandstorm requires Linux x86_64, with kernel version 3.13 or later.
 
-*Prerequisite:* Linux x86_64, with kernel version 3.13 or later.
+## Option 1: HTTPS-verified install
 
-To install on your own Linux machine, see
-[https://install.sandstorm.io/](https://install.sandstorm.io) or run:
+The easiest way to install Sandstorm is by running:
 
 ```bash
-    curl https://install.sandstorm.io | bash
+curl https://install.sandstorm.io | bash
 ```
 
-Note: If installing Sandstorm under LXC / Docker, you will need to choose the option to
-install as a non-root user. Unfortunately, this means the development tools will not
-work. This is due to the interaction between Sandstorm and Docker's use of Linux
-containerization features and missing features in the Linux kernel which we
-hope will be fixed eventually. For non-development purposes, Sandstorm should run just fine
-under Docker.
+## Option 2: Github-verified install
 
-### Tips
+If you are uncomfortable with `curl|bash`, another option is to [download install.sh from our Github repository](https://raw.githubusercontent.com/sandstorm-io/sandstorm/master/install.sh) and then run it:
 
-* If you want to run on port 80, we recommend setting up an [nginx](http://nginx.org/) reverse
-  proxy rather than trying to get Node to open port 80 directly.  Make sure to configure
-  [WebSocket forwarding](http://nginx.org/en/docs/http/websocket.html), which requires nginx
-  1.3.13 or better.
-* If you want SSL, then you will definitely need an nginx proxy (or something equivalent). You will
-  further need to use a wildcard certificate.
+```bash
+wget https://raw.githubusercontent.com/sandstorm-io/sandstorm/master/install.sh
+bash install.sh
+```
 
-For reference,
-[nginx-example.conf](https://github.com/sandstorm-io/sandstorm/tree/nginx-example.conf)
-contains the http server part of nginx config used by Sandstorm Alpha.
-## Installing from Source
+This verifies that you're running our published installer script, even in the unlikely event that someone has compromised our download server or HTTPS certificate. The installer will verify signatures on all additional files it downloads.
+
+## Option 3: PGP-verified install
+
+If you'd rather not trust HTTPS at all, even from Github, another option is PGP-verified install.
+
+1. Determine the PGP key of a Sandstorm developer you trust. There are several ways to do this:
```

Comment:
1. Find and download the PGP key of a Sandstorm developer you trust. There are several ways to do this:
    * Web of trust (for PGP experts).
    * Meet us in person and ask for our business cards.
    * Use our Keybase profiles, for example: [Kenton Varda (kentonv)](https://keybase.io/kentonv), [Asheesh Laroia (asheesh)](https://keybase.io/asheesh), [Drew Fisher (zarvox)](https://keybase.io/zarvox)

If you use the `keybase` commands to do verification, it will help you with this process, so you can skip this step.